components/alignmentannotation/README.md

```diff
@@ -2,8 +2,31 @@
 
 In the CLDF, we use space-separated strings for alignments, e.g. for [cognates](..\cognates) and possibly for [parallel texts](../../modules/ParallelText) in [FunctionalEquivalentsTable](../functionalequivalents).
 
-Such alignments are actually secondary tables within the CSV structure of the CLDF data, i.e. there are columns **inside** one such column. To be able to refer to such 'secondary' columns we use an `alignmentAnnotationTable`. One line in such a table includes the name of the table that holds an alignment columns (this name goes in the column `Alignment_ID`). Note that this is not a foreign key to a row in another table, but to a **column** in another table. The column does not have to be named, because the referenced table should only have a single column of the type [`#alignment`](http://cldf.clld.org/v1.0/terms.rdf#alignment).
+Such alignments are actually secondary tables within the CSV structure of the CLDF data, i.e. there are columns **inside** one such column. To be able to refer to such 'secondary' columns we use an `alignmentAnnotationTable`. One line in such a table includes the name of the table that holds an alignment column (this name goes in the column `Alignment_ID`). Note that this is not a foreign key to a row in another table, but to a **column** in another table. The column does not have to be named, because the referenced table should only have a single column of the type [`#alignment`](http://cldf.clld.org/v1.0/terms.rdf#alignment).
 
 Within such an alignment column we assume that the strings are aligned, so we can refer to a single column by a [`segmentSlice`](http://cldf.clld.org/v1.0/terms.rdf#segmentSlice") of an enumeration of the columns. For example, a reference to columns 3 and 4 of the alignments is simply `3:4` or `3 4`.
 
-Any annotation can be specified in the `Annotation` column. In some cases, it is practical to name what kind of annotation it is, which can be specified in the column `AnnotationKind`. For example, to make an Annotation with a standard orthography, the actual orthographic strings are the `Annotation`, but the `AnnotationKind` could specify that all these annotation are of the kind `standard orthography`.
+Any annotation can be specified in the `Annotation` column. In some cases, it is practical to name what kind of annotation it is, which can be specified in the column `AnnotationType`. For example, to make an annotation with a standard orthography, the actual orthographic strings are the `Annotation`, but the `AnnotationType` could specify that all these annotations are of the type `standard orthography`.
+
+# Example
+
+Consider a `cognates.csv` with a column `Alignment` like this:
+
+```
+Alignment
+s x o: l -
+s k u: l -
+ʃ  u: l ə
```

Comment:
> ʃ  u: l ə

should be:

```
ʃ - u: l ə
```
right?

Reply:
Well, depends how you interpret the dash, but yes, that is clearer. I'll change it